fontes/avaliador-sintatico/avaliador-sintatico.ts

```diff
@@ -280,6 +280,18 @@ export class AvaliadorSintatico implements AvaliadorSintaticoInterface {
         }
     }
 
+    private tratarValorNumerico(modificador: Simbolo): Boolean {
+        if(!(ValorNumerico.includes(modificador.lexema))) {
```

Comment:
Em que momento este método retorna `false`? Se for nunca, qual o sentido de ele existir?